tsconfig.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "@ethereumjs/config-tsc",
-  "include": ["src/**/*.ts", "test/**/*.ts"]
+  "include": ["src/**/*.ts", "test/**/*.ts", "typings/**/*.js"]
```

Comment:
I think this is typo.
```diff
- "typings/**/*.js"
+ "typings/**/*.ts"
```

Reply:
I suggest that the another approach.

`compileOptions.typeRoots`'s default value is `@types`.
So make below folders and put `ethjs-util.d.ts` to `index.d.ts`.
You don't need to modify tsconfig.json
```
- src
    - @types
        - ethjs-util
            - index.d.ts
```

Reply:
@tomonari Thanks so much for catching this! Makes sense.

Reply:
@cgewecke 
Thanks for trying it.
I also tried and find out that it is not caused of `ts` but `karma`.
I'm dealing with it.

Reply:
@cgewecke 
I made example code. Please check https://github.com/tomonari-t/ethereumjs-util/pull/2.

Reply:
@tomonari-t 

Yes that works! Have made your suggested changes in 40ecfe9 and added you as the commit author. Thank you for investigating this.